templates/template.inc.php

```diff
@@ -17,8 +17,12 @@
 	</div>
 </div>
 <?php
-    echo "<!-- Elapsed time (s): " . round((microtime(true) - $_SERVER["REQUEST_TIME_FLOAT"]), 4) . " -->\n";
-    echo "<!-- Memory usage (MB): " . round(memory_get_peak_usage(true) / (1024 * 1024), 2) . " -->\n";
+    $time   = round((microtime(true) - $_SERVER["REQUEST_TIME_FLOAT"]), 4);
+    $memory = round(memory_get_peak_usage(true) / (1024 * 1024), 2);
+    echo "<!-- Elapsed time (s): " . $time . " -->\n";
+    echo "<!-- Memory usage (MB): " . $memory . " -->\n";
+    error_log('Elapsed time (s):'  . $time);
+    error_log("Memory usage (MB): " . $memory);
 ?>
```

Comment:
I don't think we should write this in the error_log, it creates a lot of noise and make real errors harder to spot.